plugins/SimulationView/SimulationView.py

```diff
@@ -16,6 +16,7 @@
 from UM.Message import Message
 from UM.Platform import Platform
 from UM.PluginRegistry import PluginRegistry
+from UM.Qt.QtApplication import QtApplication
```

Comment:
Then you don't need to import UM.Application any more, being a parent class of QtApplication.